design/005-custodian-s3-credentials/005-custodian-s3-credentials.md

```diff
@@ -0,0 +1,65 @@
+- Start Date: 2022-04-28
+- Status: Draft
```

Comment:
I presume this is now beyond a draft? e.g. "partially implemented"?

Reply:
Yeah I will change that